ank/src/filtered_complete_state.rs

```diff
@@ -182,10 +212,35 @@ impl From<ank_base::AgentMap> for FilteredAgentMap {
             agents: Some(
                 value
                     .agents
-                    .into_keys()
-                    .map(|agent_name| (agent_name, FilteredAgentAttributes {}))
+                    .into_iter()
+                    .map(|(agent_name, agent_attributes)| (agent_name, agent_attributes.into()))
                     .collect(),
             ),
         }
     }
 }
+
+impl From<ank_base::AgentAttributes> for FilteredAgentAttributes {
+    fn from(value: ank_base::AgentAttributes) -> Self {
+        FilteredAgentAttributes {
+            cpu_load: value.cpu_load.map(Into::into),
```

Comment:
Do we need the map() here?, won't a plain .into() just work without the additional call?